code/modules/mob/new_player/new_player.dm

```diff
@@ -97,6 +97,13 @@
 					var/mob/dead/observer/O = i
 					stat("[O.client?.holder?.fakekey ? O.client.holder.fakekey : O.key]", "Observing")
 
+
+/mob/new_player/can_interact_with(datum/D)
+	if(D != src)
```

Comment:
Are you sure this is intended? Could you explain what this proc does?

Reply:
New player has their own topic, this makes them able to interact with only themselves.